Added enemies on platforms

ArlanGmz opened the pull request:
Please comment

ArlanGmz committed:
added enemy on platforms

ArlanGmz committed:
edit of smaller platform

nmvNoble reviewed (changes requested):
Enemies work fine but sometimes spawn at the starting edge of a platform that is too high for the player to jump past it there fore making some instances impossible to pass.

ArlanGmz committed:
Edited Enemy and platform spawning

Enemy will now not spawn on first platform and if a platform with an enemy is too high, platform will be set lower
